src/serac/physics/tests/test_solid_functional_statics.cpp

```diff
@@ -0,0 +1,290 @@
+// Copyright (c) 2019-2022, Lawrence Livermore National Security, LLC and
+// other Serac Project Developers. See the top-level LICENSE file for
+// details.
+//
+// SPDX-License-Identifier: (BSD-3-Clause)
+
+#include "serac/physics/solid_functional.hpp"
+
+#include <functional>
+#include <set>
+#include <string>
+
+#include "axom/slic/core/SimpleLogger.hpp"
+#include <gtest/gtest.h>
+#include "mfem.hpp"
+
+#include "serac/mesh/mesh_utils.hpp"
+#include "serac/physics/state/state_manager.hpp"
+#include "serac/physics/materials/solid_functional_material.hpp"
+#include "serac/serac_config.hpp"
+
+namespace serac {
+
+using solid_mechanics::direct_static_options;
+
+/**
+ * @brief Exact displacement solution that is an affine function
+ *
+ * @tparam dim number of spatial dimensions
+ */
+template <int dim>
+class AffineSolution {
+ public:
+  AffineSolution():
+    A(dim), b(dim)
+  {
+    // clang-format off
+    A(0, 0) = 0.110791568544027; A(0, 1) = 0.230421268325901;
+    A(1, 0) = 0.198344644470483; A(1, 1) = 0.060514559793513;
+    if constexpr (dim == 3) {
+      A(0, 2) = 0.15167673653354;
+      A(1, 2) = 0.084137393813728;
+      A(2, 0) = 0.011544253485023; A(2, 1) = 0.060942846497753; A(2, 2) = 0.186383473579596;
+    }
+
+    b(0) = 0.765645367640828;
+    b(1) = 0.992487355850465;
+    if constexpr (dim == 3) {
+      b(2) = 0.162199373722092;
+    }
+    //clang-format on
+  };
+
+  /**
+   * @brief MFEM-style coefficient function corresponding to this solution
+   *
+   * @param X Coordinates of point in reference configuration at which solution is sought
+   * @param u Exact solution evaluated at \p X
+   */
+  void operator()(const mfem::Vector& X, mfem::Vector& u) const
+  {
+    u = 0.0;
+    A.Mult(X, u);
```

Comment:
I think `Operator::Mult()` zeroes out the output-vector for us (?), which would make line 62 is unnecessary

Reply:
Thanks. You're right. Fixed.